qml/pages/SettingsPage.qml

```diff
@@ -64,6 +64,40 @@ Page {
                 text: qsTrId("chum-settings-testing")
                 onClicked: Chum.repoTesting = !Chum.repoTesting;
             }
+
+            SectionHeader {text: qsTrId("chum-settings-advanced")}
+
+            Label {
+                anchors {
+                    left: parent.left
+                    leftMargin: Theme.horizontalPageMargin
+                    right: parent.right
+                    rightMargin: Theme.horizontalPageMargin
+                }
+                color: Theme.highlightColor
+                text: qsTrId("chum-settings-override-release")
```

Comment:
chum-settings-override-release is missing a string

Reply:
I would suggest to use a string that you had here "Override Chum Repository Release" . Not sure about capitalization as I haven't used it in other places. However, we do need more info to know what to enter, see below in TextField

Reply:
Ah ok, i didnt know how these worked!